src/legacy/core_plugins/kibana/public/visualize/wizard/type_selection/type_selection.tsx

```diff
@@ -152,7 +152,9 @@ class TypeSelection extends React.Component<TypeSelectionProps, TypeSelectionSta
                     </h2>
                   </EuiTitle>
                   <EuiSpacer size="m" />
-                  <NewVisHelp />
+                  <NewVisHelp
+                    promotedTypes={(visTypes as VisTypeAliasListEntry[]).filter(t => t.promotion)}
```

Comment:
1. This could return multiple promotions?
2. This assumes the promotions are alias types, which I've commented on elsewhere